From cfa7c83bca6ad927b3168386fc31d7a12d93cd13 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:52 -0400
Subject: [tools] perf/stat: Remove misplaced __maybe_unused

Message-id: <1408953210-26343-225-git-send-email-jolsa@redhat.com>
Patchwork-id: 88896
O-Subject: [PATCH RHEL7.1 BZ1133083 224/322] perf stat: Remove misplaced __maybe_unused
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 41cde47675de62ee0f3877c00ab44373b2b2f4ca
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Fri Jan 3 17:34:42 2014 -0300

description
===========
That 'argc' argument _is_ being used.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-stat.c b/tools/perf/builtin-stat.c
index 6ca0766..b27b264 100644
--- a/tools/perf/builtin-stat.c
+++ b/tools/perf/builtin-stat.c
@@ -645,7 +645,7 @@ static int __run_perf_stat(int argc, const char **argv)
  return WEXITSTATUS(status);
 }
 
-static int run_perf_stat(int argc __maybe_unused, const char **argv)
+static int run_perf_stat(int argc, const char **argv)
 {
  int ret;
 
-- 
1.7.1